akka-docs/rst/java/logging.rst

```diff
@@ -123,6 +123,19 @@ If you want very detailed logging of all lifecycle changes of Actors (restarts,
       }
     }
 
+If you want unhandled messages logged at DEBUG (instead of being sent on the event bus as :class:`akka.actor.UnhandledMessage`):
```

Comment:
Ah, I was pretty sure that is how it is but then one of the other doc locations threw me off. I'll double check that in addition to fixing this, in case it wasn't just me that misread.